Documentation/dev_workflow.md

```diff
@@ -0,0 +1,156 @@
+This document shows an example of development process for Xen and GRUB installed
+on remote machine.
```

Comment:
Instructions in this document doesn't depend on specific hardware, and firmware limitation is just "not UEFI". I don't feel comfortable advertising firmware that isn't even able to reboot half of the times.

Reply:
@krystian-hebel nobody ask you for advertising, without those information there is no way to reproduce your work. Please provide information we asking for.

Reply:
> doesn't depend on specific hardware

Can I use with AMD hardware? Can I use with Intel hardware with no TXT? A single bullet point explaining what hardware we use for testing would not hurt.

> and firmware limitation is just "not UEFI"

And TXT support properly enabled. And proper ACM included. And maybe more, which I'm not aware of.

Why not just at least put these requirements you have mentioned in the document, rather than in comment? Wouldn't it be more useful for the reader (and also quicker)?